packages/react-button/src/components/ToggleButton/useToggleButton.ts

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+import { useControllableValue } from '@uifabric/react-hooks';
+import { ToggleButtonProps } from './ToggleButton.types';
+
+/**
+ * The useToggleButton hook processes the Button component props and returns state.
+ * @param props
+ */
+export const useToggleButton = (props: ToggleButtonProps) => {
```

Comment:
input props should be ButtonProps type. pls also add return type

Reply:
Why should it be `ButtonProps`?

I'll add the return type.